README.md

```diff
@@ -22,6 +22,9 @@ https://anuvu.github.io/zot/
 * Storage optimizations:
   * Automatic garbage collection of orphaned blobs
   * Layer deduplication using hard links when content is identical
+* Multiple storage backend using single zot server:
+  * Added subpath sections in storage config, see example config [file](./examples/config-multiple.json) 
```

Comment:
config-multiple.json <- where is this file, not in this PR?

Reply:
Its in the PR